iceoryx_platform/win/include/iceoryx_platform/mqueue.hpp

```diff
@@ -52,12 +52,12 @@ inline int mq_close(mqd_t mqdes)
 //{
 //}
 
-inline ssize_t mq_receive(mqd_t mqdes, char* msg_ptr, size_t msg_len, unsigned int* msg_prio)
+inline mqd_t mq_receive(mqd_t mqdes, char* msg_ptr, size_t msg_len, unsigned int* msg_prio)
 {
     return 0;
 }
 
-inline ssize_t
+inline mqd_t
```

Comment:
This should also not be required

Reply:
@elBoberido there will also be an error, without this change

Reply:
mingw or msvc?

With the latest changes this should also be fixed.